engine/src/org/pentaho/di/trans/TransMeta.java

```diff
@@ -2878,7 +2880,7 @@ public void loadXML( Node transnode, String fname, IMetaStore metaStore, Reposit
             }
           }
         }
-        setPrivateTransformationDatabases( privateTransformationDatabases );
+        setPrivateDatabases( privateTransformationDatabases );
```

Comment:
Why replace this call with the AbstractMeta version, rather than just have setPrivateTransformationDatabases (the existing method) call the AbstractMeta version?  

With the fix above, what happens if some external code calls setPrivateTransformationDatabases()? The TransMeta's privateTransformationDatabases would be set but its AbstractMeta's privateDatabases would not be. The inverse appears true as well (if setPrivateDatabases() is called then getPrivateTransformationDatabases() is called)